recurring_contract/models/contract_group.py

```diff
@@ -485,9 +524,11 @@ def _build_invoice_gen_data(self, invoicing_date, invoicer, gift_wizard=False):
                         invoicing_date=invoicing_date, contract_line=cl
                     ),
                 )
+                # This loop is the cause in invoices being doubled/tripled/...
+                # We should only get here if there is not an existing open invoice for this period,
+                # so before there was no reason to reference the contract_line_ids of the other active invoices
                 for cl in (
```

Comment:
@ecino I think this is caused the issue of  https://github.com/CompassionCH/compassion-accounting/pull/233 of multiple invoices for the same invoice date that were generated.

This code is only executed if we have no open invoice for that month, but since it then looped on active_contract_ids.contract_line_ids, it would generate an invoice for every active contract, so 2 if there are 2 contract, 3 if 3...